lib/themes/dosomething/paraneue_dosomething/js/reportback/Reportback.js

```diff
@@ -273,7 +278,33 @@ const Reportback = {
         }
       },
     }).done(function(response) {
-      _this.apiUrl = response.meta.pagination.links.next_uri || null;
+      // Check to see if the response if from Rogue.
+      if (Drupal.settings.dsRogue.poweringGallery) {
```

Comment:
Hmm, yeah I'm curious if we can pass along the proper base URL so Rogue can add that to the original request (maybe via a custom header?) but this is definitely good to merge in the meantime!